src/linker/Linker.Dataflow/FlowAnnotations.cs

```diff
@@ -168,7 +168,8 @@ private TypeAnnotations BuildTypeAnnotations (TypeDefinition type)
 					MethodDefinition setMethod = property.SetMethod;
 					if (setMethod != null) {
 
-						if (!ScanMethodBodyForFieldAccess (setMethod.Body, write: true, out backingFieldFromSetter)) {
+						// TODO: Handle abstract properties - no way to propagate the annotation to the field
```

Comment:
Do we really need to do anything with the TODO here? I think this will fall out once we enforce that all virtuals have matching annotations - the property will be reintroduced in the derived class and the setter will be a virtual override of the abstract base.

Reply:
I think so as well, but I wanted to add the TODO so that we don't forget to validate that.